docs/troubleshooting/index.mdx

```diff
@@ -35,6 +37,61 @@ Alpine is missing some dependencies that [are required](https://github.com/datre
 Adding the following command to the CI should fix this: `apk add curl bash`
 This will add curl and bash commands which our install scripts require in order to run.
 
+---
+
+## Schema validation failures
+
+If you get the following message in your CLI: 
+```terminal
+❌  k8s schema validation error: could not find schema for [KIND]  
+``` 
+It means that the resource's schema which is used to perform the schema valiation was not found, and it can be caused by any of the following-
+
+### Wrong K8s version is used in the schema validation check
+To perform the schema validation check, the CLI pulls the relevant schema according to your default K8s schema, or the value of the `--schema-version` flag.  
+To fix this failure, update your default K8s version in the dashboard to the same K8s version as your cluster, or pass the `--schema-version` flag.  
+Instructions for how to update your default K8s version can be found [here](https://hub.datree.io/setup/schema-validation#changing-the-default-schema-version)
+
+### Missing schema for a CRD
+As part of the schema validation phase, the CLI looks for the schemas in multiple GitHub repositories that contain the schema of native K8s objects and popular CRDs.  
+If you're testing a CRD that is neither a native K8s object, nor listed in the CRDs catalog, you can extract the schema to validate your manifests.  
+Find the instructions [here](https://hub.datree.io/setup/schema-validation#crd-support)
+
+### Deprecated APIversion
+Some resources' API version got deprecated in the latest k8s version releases, it could be that you should migrate your resources to the newer version. 
```

Comment:
"latest k8s version releases" - once a new version is released, this sentence won't be relevant anymore...